### DISCUSSION TITLE
[灌水]有谁认识这个狼希人？

### DISCUSSION THREAD

**Discussion by nninng** [share121/inter-knot#2846] [其他]
http://d.0wd.cn/3cSiE8
今天五六点钟左右在光映广场逛着，想去报刊买份今天的报纸晚上看，看到光映广场的江边步道边站着一位狼希人，看起来好像在等人的样子，有点小帅，所以偷偷拍了，有谁认识吗？（如果侵权我会删除此贴）

**Comment by aizai1173** [comment]
是我老公在等我呢（^-^）
